test/fixture.cpp

```diff
@@ -410,3 +399,16 @@ void TestFixture::setTemplateFormat(const std::string &templateFormat)
         mTemplateLocation = "";
     }
 }
+
+TestFixture::SettingsBuilder& TestFixture::SettingsBuilder::library(const char lib[]) {
+    // TODO: exename is not yet set
+    LOAD_LIB_2_EXE(settings.library, lib, fixture.exename.c_str());
+    // strip extension
+    std::string lib_s(lib);
+    static const std::string ext(".cfg");
```

Comment:
There is no reason ext must be static. I feel skeptic about it.

Reply:
Yeah - no idea why I did that. Must have been a left-over from an earlier revision. I went froth and back on the implementation.

Reply:
Done